### PR TITLE
Add Nimbus and Erigon guide

### PR DESCRIPTION
**Pull request by zengzengzenghuy**
## What
* Add Nimbus **docker-compose.yml** configuration
* Add Erigon CLI guide

### Nimbus
* https://github.com/gnosischain/gnosis-nimbus-eth2
* https://nimbus.guide/
### Erigon
* https://github.com/ledgerwatch/erigon#documentation

### CODE REVIEW

File: docs/node/guide/beacon/_partials/_install_cl_nimbus.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+<TabItem value="gnosis">
+
+<Tabs className="tabgroup-with-label os-tabgroup" groupId="os" defaultValue="others" values={[
+{label: 'Linux, MacOS, Arm64', value: 'others'},
+{label: 'Windows', value: 'win'}
+]}>
+<TabItem value="others">
+
+:::danger
+Nimbus is not ready for production use. Currently, you can only use Docker to run Nimbus Beacon Node + Validator.
```
Comment (filoozom) [inline docs/node/guide/beacon/_partials/_install_cl_nimbus.md:18]:
```suggestion
:::caution
Nimbus is considered experimental for Gnosis and only supports Docker deployments for now. For production ready alternatives, please consider Lighthouse, Teku or Lodestar.
```

File: docs/node/guide/execution/_partials/_install_el_erigon.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+]}>
+<TabItem value="gnosis">
+<Tabs className="tabgroup-with-label os-tabgroup" groupId="os" defaultValue="others" values={[
+{label: 'Linux, MacOS, Arm64', value: 'others'},
+{label: 'Windows', value: 'win'}
+]}>
+<TabItem value="others">
+
+:::danger
+[Erigon](https://github.com/ledgerwatch/erigon#documentation) is not ready for production use. Please consider switching to Nethermind.
```
Comment (filoozom) [inline docs/node/guide/execution/_partials/_install_el_erigon.md:16]:
```suggestion
:::caution
[Erigon](https://github.com/ledgerwatch/erigon#documentation) is considered experimental for Gnosis. For production use, please consider Nethermind.
```

**Review by filoozom — COMMENTED** [review]
A few nitpicks, but in general great work! Looking forward to this shipping. 😁

File: docs/node/guide/beacon/nimbus.md [hunk 3]
```diff
@@ -22,28 +23,124 @@ Nimbus is a client implementation that strives to be as lightweight as possible
 :::info 
 
 - Gnosis' Nimbus repo has sample Dockerfiles and configs
-- [https://github.com/gnosischain/nimbus-client](https://github.com/gnosischain/nimbus-client)
+- [https://github.com/gnosischain/gnosis-nimbus-eth2](https://github.com/gnosischain/gnosis-nimbus-eth2)
```
[the remark this answers is not in the document]
Comment (filoozom) [reply docs/node/guide/beacon/nimbus.md:26]: My bad, that's indeed correct 😅

File: docs/node/guide/execution/erigon.md [hunk 4]
```diff
@@ -9,5 +9,13 @@ Formerly TurboGeth, Erigon is an Ethereum client built to enable performance opt
...
 :::note
-Erigon is currently working on a Gnosis implementation, please, use [Nethermind](./nethermind.md)
-:::
\ No newline at end of file
+Erigon is not ready for production use, please consider switching to [Nethermind](./nethermind.md)
+:::
+
+## Option 1: Using Docker
+
+WIP
```
[the remark this answers is not in the document]
Comment (zengzengzenghuy) [reply docs/node/guide/execution/erigon.md]: Done in [cadbb26](https://github.com/gnosischain/documentation/pull/408/commits/cadbb26f053e3a218efe534cee5c44210ebca9c4)

**Review by filoozom — CHANGES_REQUESTED** [review]
The Nimbus docs look good, for the Erigon ones I would like to have them a bit more like the Nethermind ones. Currently, it's missing a few steps that would make it incompatible with how the way the CL docs are written.

At least, we should add:
- The folder structure
- The `version` in `docker-compose.yaml`
- The `gnsosi_net` network
- All the other steps
  - Archive node vs pruned node
  - JWT Secret
  - Start container
  - Monitor logs
  - Updating your node

We should also change:
- The volume paths (`/home/$USER/gnosis/execution`, same for the jwt)

File: docs/node/guide/execution/erigon.md [hunk 5]
```diff
@@ -12,19 +12,40 @@ Repository: [https://github.com/ledgerwatch/erigon](https://github.com/ledgerwat
...
+
+Create a docker-compose file with your favorite text editor in `/home/$USER/gnosis/docker-compose.yml`:
+
+```shell title="/home/$USER/gnosis/docker-compose.yml"
+version: "3"
 services:
-  erigon:
+  execution:
     image: thorax/erigon:devel
     restart: unless-stopped
```
Comment (filoozom) [inline docs/node/guide/execution/erigon.md:45]: Not sure why I can't suggest changes anymore, but we're missing the network here:
```
        networks:
          - gnosis_net
```

File: docs/node/guide/execution/erigon.md [hunk 6]
```diff
@@ -24,10 +24,13 @@ Create your folder structure:
 
 ```shell
 mkdir -p /home/$USER/gnosis/jwtsecret
+cd /home/$USER/gnosis
+mkdir ./data && sudo chown 1000:1000 ./data
```
[the remark this answers is not in the document]
Comment (zengzengzenghuy) [reply docs/node/guide/execution/erigon.md]: Change `/data` to `/execution` [e34d0a4](https://github.com/gnosischain/documentation/pull/408/commits/e34d0a472025bb4c53137ac105bdb65a7dde1a43)

**Review by filoozom — APPROVED** [review]

**Pull request merged**